.github/workflows/benchmarks.yml

```diff
@@ -0,0 +1,28 @@
+---
+    name: benchmark tests
+    
+    on:
+      workflow_dispatch:
+    
+    jobs:
+      build:
+        name: Benchmark Tests
+        runs-on: ubuntu-latest-16-cores
```

Comment:
Do we need to run on larges?

Reply:
My thinking is that the shared runners will have more variance due to other workloads potentially running at the same time. The larger runners might be more consistent, but I'm not actually sure.